### ISSUE TITLE
Teaching tip narrator announcement does not announce the subtitle

### ISSUE THREAD

**Issue by nilaysanghvi** [microsoft/microsoft-ui-xaml#3277]
The current narrator announcement is - "Press F6 to go to new notification from [App name]. [Teaching tip Title]". 
Ideally, the subtitle should be announced too, and if not then at least the developer should be given the option to control if the subtitle should be announced or not.

**Comment by StephenLPeters** [comment]
I'm not sure that I agree that the subtitle should always be read along with the teaching tip title. However you should be able to adjust what narrator reads by setting automation properties on your TeachingTip.  Setting AutomationID of the tip eventually gets folded into the announcement from narrator here: https://github.com/microsoft/microsoft-ui-xaml/blob/70adca81f36a1828adce6f9a9ab92892f4eeb9a9/dev/TeachingTip/TeachingTip.cpp#L1203.  It is possible we should be folding more automation properties into this announcement, but does this point of customization fulfill your needs?

**Comment by nilaysanghvi** [comment]
@StephenLPeters Yes it does. Thanks a lot!
